File: FrontEnd/src/app/features/reservations/schedules/classes/form/schedule-read-dto.ts
import { DestinationAutoCompleteVM } from '../../../destinations/classes/view-models/destination-autocomplete-vm'
import { Metadata } from 'src/app/shared/classes/metadata'
import { PortAutoCompleteVM } from '../../../ports/classes/view-models/port-autocomplete-vm'

export interface ScheduleReadDto extends Metadata {

    // PK
    id: number
    // Object fields
    destination: DestinationAutoCompleteVM
    port: PortAutoCompleteVM
    // Fields
    date: string
    maxPax: number
    time: string
    isActive: boolean

}

File: API/Features/Reservations/Schedules/Dtos/ScheduleReadDto.cs
using API.Infrastructure.Classes;
using API.Infrastructure.Interfaces;

namespace API.Features.Reservations.Schedules {

    public class ScheduleReadDto : IBaseEntity, IMetadata {

        // PK
        public int Id { get; set; }
        // Fields
        public string Date { get; set; }
        public int MaxPax { get; set; }
        public string Time { get; set; }
        public bool IsActive { get; set; }
        // Metadata
        public string PostAt { get; set; }
        public string PostUser { get; set; }
        public string PutAt { get; set; }
        public string PutUser { get; set; }
        // Navigation
        public SimpleEntity Destination { get; set; }
        public SimpleEntity Port { get; set; }

    }

}